SQL/01_create_peeps_table.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TABLE peep(id SERIAL PRIMARY KEY, body VARCHAR(280), maker_id INTEGER);
+CREATE TABLE users(id SERIAL PRIMARY KEY, email VARCHAR(60), password VARCHAR(140));
```

Comment:
I wasn't sure about creating 2 separate tables but this seems like a logical way to go about things.